content/en/_index.html

```diff
@@ -12,7 +12,7 @@
 	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="docs/getting-started">
 		Get Started <i class="fab fa-github ml-2 "></i>
 	</a>
-	<p class="lead mt-5">Let's explore Merbridge</p>
+	<p class="lead mt-5">Well supports Istio, Linkerd2, and Kuma.</p>
```

Comment:
```suggestion
	<p class="lead mt-5">Currently supported Istio, Linkerd2, and Kuma.</p>
```

Reply:
`currrently` is a bit contradictory with `-ed`, how about this one:
```
Istio, Linkerd2, and Kuma are supported currently.
```

Reply:
`Currently supported sth` and `sth is currently supported` are basically the same thing.

Reply:
Well, modified. Shorter is better here.